pkg/synthetictests/duplicated_events.go

```diff
@@ -199,13 +237,94 @@ func (d duplicateEventsEvaluator) testDuplicatedEvents(testName string, flakeOnl
 
 		if flake || flakeOnly {
 			flakes = append(flakes, appendToFirstLine(msg, " result=allow "))
+			fmt.Println("Found patho flake")
 		} else {
+			fmt.Println("Found patho failure")
 			failures = append(failures, appendToFirstLine(msg, " result=reject "))
 		}
+
+		pathoEvent := pathoItem.pathologicalEvent
+		fmt.Println("Checking pathoEvent.Message: ", pathoEvent.Message)
+
+		if strings.Contains(pathoEvent.Message, duplicateevents.PathologicalMark) && !strings.Contains(pathoEvent.Message, duplicateevents.InterestingMark) {
+
+			fmt.Println("Processing event with pathologicalTrue but no interesting mark")
+
+			// This event occurred more than threshold but is not recognized (i.e., unknown).
+			// So we need to mark all other instances of this event so we can chart when this
+			// event started happening.
+
+			clientSet, clientErr := kubernetes.NewForConfig(kubeClientConfig)
```

Comment:
We should not be talking live to a cluster for this, we have all we need in the event intervals already I think. Will explain more below.

Reply:
Thanks for this.  Yes, I agree if we have what we need there already, no need to get it from the cluster.